[09:51:13.260] [INFO ] - Clicking on element [Main]... (DeviceElementActions:) 
[09:51:13.343] [ERROR] - com.github.wasiqb.coteafs.appium.error.DeviceElementNameNotFoundError: DeviceElement with name [GetStarted] not found. (ErrorUtils:) 
[09:51:13.344] [ERROR] - 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) (ErrorUtils:) 
[09:51:13.344] [ERROR] - 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) (ErrorUtils:) 
[09:51:13.344] [ERROR] - 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) (ErrorUtils:) 
[09:51:13.345] [ERROR] - 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) (ErrorUtils:) 
[09:51:13.345] [ERROR] - 	at com.github.wasiqb.coteafs.error.util.ErrorUtil.fail(ErrorUtil.java:102) (ErrorUtils:) 
[09:51:13.345] [ERROR] - 	at com.github.wasiqb.coteafs.error.util.ErrorUtil.fail(ErrorUtil.java:70) (ErrorUtils:) 
[09:51:13.345] [ERROR] - 	at com.github.wasiqb.coteafs.appium.utils.ErrorUtils.fail(ErrorUtils.java:53) (ErrorUtils:) 
[09:51:13.346] [ERROR] - 	at com.github.wasiqb.coteafs.appium.utils.ErrorUtils.fail(ErrorUtils.java:40) (ErrorUtils:) 
[09:51:13.346] [ERROR] - 	at com.github.wasiqb.coteafs.appium.device.DeviceActivity.getDeviceElement(DeviceActivity.java:211) (ErrorUtils:) 
[09:51:13.346] [ERROR] - 	at com.github.wasiqb.coteafs.appium.device.DeviceActivity.getElement(DeviceActivity.java:99) (ErrorUtils:) 
[09:51:13.346] [ERROR] - 	at com.github.wasiqb.coteafs.appium.android.AndroidActivity.onElement(AndroidActivity.java:68) (ErrorUtils:) 
[09:51:13.347] [ERROR] - 	at com.jtv.tests.getStartedTest.testGetStarted(getStartedTest.java:33) (ErrorUtils:) 
[09:51:13.347] [ERROR] - 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) (ErrorUtils:) 
[09:51:13.347] [ERROR] - 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) (ErrorUtils:) 
[09:51:13.347] [ERROR] - 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) (ErrorUtils:) 
[09:51:13.348] [ERROR] - 	at java.lang.reflect.Method.invoke(Method.java:498) (ErrorUtils:) 
[09:51:13.348] [ERROR] - 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108) (ErrorUtils:) 
[09:51:13.348] [ERROR] - 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661) (ErrorUtils:) 
[09:51:13.349] [ERROR] - 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869) (ErrorUtils:) 
[09:51:13.349] [ERROR] - 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193) (ErrorUtils:) 
[09:51:13.349] [ERROR] - 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126) (ErrorUtils:) 
[09:51:13.349] [ERROR] - 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109) (ErrorUtils:) 
[09:51:13.350] [ERROR] - 	at org.testng.TestRunner.privateRun(TestRunner.java:744) (ErrorUtils:) 
[09:51:13.350] [ERROR] - 	at org.testng.TestRunner.run(TestRunner.java:602) (ErrorUtils:) 
[09:51:13.350] [ERROR] - 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380) (ErrorUtils:) 
[09:51:13.350] [ERROR] - 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375) (ErrorUtils:) 
[09:51:13.351] [ERROR] - 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340) (ErrorUtils:) 
[09:51:13.351] [ERROR] - 	at org.testng.SuiteRunner.run(SuiteRunner.java:289) (ErrorUtils:) 
[09:51:13.351] [ERROR] - 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) (ErrorUtils:) 
[09:51:13.351] [ERROR] - 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86) (ErrorUtils:) 
[09:51:13.352] [ERROR] - 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301) (ErrorUtils:) 
[09:51:13.352] [ERROR] - 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226) (ErrorUtils:) 
[09:51:13.352] [ERROR] - 	at org.testng.TestNG.runSuites(TestNG.java:1144) (ErrorUtils:) 
[09:51:13.352] [ERROR] - 	at org.testng.TestNG.run(TestNG.java:1115) (ErrorUtils:) 
[09:51:13.353] [ERROR] - 	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:178) (ErrorUtils:) 
[09:51:13.353] [ERROR] - 	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92) (ErrorUtils:) 
[09:51:13.353] [ERROR] - 	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:96) (ErrorUtils:) 
[09:51:13.353] [ERROR] - 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) (ErrorUtils:) 
[09:51:13.354] [ERROR] - 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) (ErrorUtils:) 
[09:51:13.354] [ERROR] - 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) (ErrorUtils:) 
[09:51:13.354] [ERROR] - 	at java.lang.reflect.Method.invoke(Method.java:498) (ErrorUtils:) 
[09:51:13.354] [ERROR] - 	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:208) (ErrorUtils:) 
[09:51:13.354] [ERROR] - 	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:159) (ErrorUtils:) 
[09:51:13.355] [ERROR] - 	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:87) (ErrorUtils:) 
[09:51:13.355] [ERROR] - 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153) (ErrorUtils:) 
[09:51:13.355] [ERROR] - 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:95) (ErrorUtils:) 
